checks/containers/container_engine/xccl_tests.py

```diff
@@ -127,26 +143,27 @@ def setup_ce(self):
 
 
 @rfm.simple_test
-class RCCLTestCE(XCCLTestBase):
+class NCCLTestsUENV(XCCLTestBase):
```

Comment:
Does it make sense to change the test names to be a bit more descriptive?

Reply:
Do you have an example of what you'd prefer as a more descriptive name? Add something with `perf` or `performance`? Or something else?